src/plugins/lua/antivirus.lua

```diff
@@ -162,18 +162,24 @@ local function sophos_config(opts)
   return nil
 end
 
+local function message_too_large(task, rule)
+  local max_size = tonumber(rule['max_size'])
+  if not max_size then return end
+  if task:get_size() > max_size then return true end
```

Comment:
That's a bit inconsistent: you return 3 values from a function: `true`, `false` and `nil`. I think it's not a good idea.